### PR TITLE
Added image picker and emoji picker dark icons for light theme

### PR DESCRIPTION
**Pull request by manny525**
The image picker and emoji picker icons where not visible in the light theme.

### CODE REVIEW

File: client/src/components/Messaging/ImagePicker/index.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import React from 'react';
 import imagePicker from '../../../utils/imagePicker.js';
+import { ThemeContext } from '../../ThemeContext.js';
```
Comment (muke1908) [inline client/src/components/Messaging/ImagePicker/index.js:3]: The build is failing here.

**Review by muke1908 — CHANGES_REQUESTED** [review]

**Comment by muke1908** [comment]
@all-contributors, please add @manny525 for code.

**Comment by muke1908** [comment]
@all-contributors please add @manny525 for code.

**Comment by manny525** [comment]
@muke1908 do you mind adding the hacktoberfest-accepted label to this PR?

**Pull request merged**